src/exec/DB.hs

```diff
@@ -86,7 +83,9 @@ withDB dbOptions dbTracer indexTracer rr btime nodeConfig extLedgerState k = do
       cdbDiskPolicy = DiskPolicy
         { onDiskNumSnapshots  = 2
           -- Take a snapshot every 20s
-        , onDiskWriteInterval = return $ secondsToDiffTime 20
+        , onDiskShouldTakeSnapshot = \mTime _ -> case mTime of
+            Nothing -> True
+            Just it -> it >= secondsToDiffTime 20
```

Comment:
Every 20s is quite often, especially when a proxy is fully synced (a ledger snapshot of a fully synced chain is +62 MB!).
We have recently tuned the default policy to work better in all cases, so there's no need to override it anymore.